llvm/test/tools/sycl-post-link/sycl-grf-size.ll

```diff
@@ -0,0 +1,72 @@
+; This test checks handling of sycl-grf-size in SYCL post link
+
+; RUN: sycl-post-link -split=source -symbols -split-esimd -lower-esimd -S < %s -o %t.table
+; RUN: FileCheck %s -input-file=%t.table
+; RUN: FileCheck %s -input-file=%t_esimd_0.ll --check-prefixes CHECK-ESIMD-LargeGRF-IR --implicit-check-not='__ESIMD_kernel()'
```

Comment:
Thank you for doing this additional fix here.
Wouldn't it be useful to do something similar for __SYCL_kernel() ? I.e. check that it doesn't get copied to SYCL-LargGrf-IR

Reply:
yeah sure, i just pushed a commit to test all the cases here to make sure we don't end up with the wrong grf-mode in a split